views/templates/admin/tabs/help.tpl

```diff
@@ -0,0 +1,94 @@
+{*
+* 2007-2018 PrestaShop
+*
+* DISCLAIMER
+*
+* Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+* versions in the future. If you wish to customize PrestaShop for your
+* needs please refer to http://www.prestashop.com for more information.
+*
+* @author    PrestaShop SA <contact@prestashop.com>
+* @copyright 2007-2018 PrestaShop SA
+* @license   http://addons.prestashop.com/en/content/12-terms-and-conditions-of-use
+* International Registered Trademark & Property of PrestaShop SA
+*}
+
+<div class="panel col-lg-10 right-panel">
+    <h3>
+        <i class="fa fa-question-circle"></i> {l s='Help & contact' mod='ps_faviconnotificationbo'} <small>{$module_display|escape:'htmlall':'UTF-8'}</small>
```

Comment:
Can you confirm that it is the 's=' that introduces translation keys here? If it is the case, we must localize it with the proper domains, I can help you with that if needed. ;-)

Reply:
I have made some changes on my last commit, could you please check this and tell me if my changes are okay ?

Reply:
totally deleted